test/test_helper.rb

```diff
@@ -31,7 +31,7 @@ class SubclassGateway < SimpleTestGateway
 
 module ActiveMerchant
   module Assertions
-    AssertionClass = defined?(Minitest) ? MiniTest::Assertion : Test::Unit::AssertionFailedError
+    @assertion_class = defined?(Minitest) ? MiniTest::Assertion : Test::Unit::AssertionFailedError
```

Comment:
This actually subtly changes this entity from a constant to a class-level variables, which makes it mutable. Given how it is used, I don't _think_ this matters, but what do you think about changing `AssertionClass` to `ASSERTION_CLASS` in order to make Rubocop happy and keep it as a constant just in case?

Reply:
Thanks for the suggestion! That does indeed appear to appease the mighty RuboCop. 🎉